internal/app/wwctl/node/list/main_test.go

```diff
@@ -602,6 +602,29 @@ nodes:
   n02:
     profiles:
     - default
+`,
+		},
+		{
+			name:    "single node list with network",
+			args:    []string{"-a"},
+			wantErr: false,
+			stdout: `
+NODE  FIELD                    PROFILE  VALUE
+----  -----                    -------  -----
+n01   Profiles                 --       default
+n01   NetDevs[default].Hwaddr  --       aa:bb:cc:dd:ee:ff
+n01   NetDevs[default].Ipaddr  --       1.1.1.1
```

Comment:
Not sure if is must exactly look like that, but no commas and additional zeros should be in here.